src/libraries/System.Linq/tests/AnyTests.cs

```diff
@@ -63,6 +63,19 @@ public void Any(IEnumerable<int> source, bool expected)
             Assert.Equal(expected, source.Any());
         }
 
+        [Fact]
+        public void Any_Grouping()
+        {
+            var grouppedValues = new int[] { 1, 2 }.GroupBy(num => num);
+            Assert.True(grouppedValues.Any());
+
+            var grouppedValuesToEmpty = string.Empty.GroupBy(ch => ch);
```

Comment:
Fundamentally, the different element types don't change what is being tested. Would recommend just using one type (`int` for simplicity).

Reply:
OK, you like that style....I'll fix it ASAP.